### ISSUE TITLE
Multiple configuration files found.

### ISSUE THREAD

**Issue by sanglnv** [kentcdodds/testing-workshop#106]
Hi there, 
I faced with "Multiple configuration files found" issue because of the definition of babel in both package.json and .babelrc.js while i run test without command "npm run setup --silence", any magic here to fixed this issue after tried setup command? 
Regards,

**Comment by kentcdodds** [comment]
Hi @sanglnv, I'm unable to reproduce this. Which branch are you running your setup command on?

**Comment by kentcdodds** [comment]
I've made a few changes to the repo that should avoid this problem. Please let me know if you see it again.

**Comment by nekhaevskiy** [comment]
I had the same problem when I just repeated @kentcdodds actions in the course and install Jest using command `npm install --save-dev jest` because it installs latest version of Jest. After I manually installed version 22.4.3 the problem was gone. 
But I had another one:
```
> jest

 FAIL  src\__tests__\utils.js
  ● Test suite failed to run

    SecurityError: localStorage is not available for opaque origins

      at Window.get localStorage [as localStorage] (node_modules/jsdom/lib/jsdom/browser/Window.js:257:15)
          at Array.forEach (<anonymous>)
```
This problem was solved by installing jest 23.6.0
